KidtronicsLanguage/var.cpp

```diff
@@ -10,13 +10,13 @@
 #include <string.h>
 #include <stdlib.h>
 #include "var.h"
+#include <iostream>
```

Comment:
I don't think you can compile this library with arduino.

Reply:
It was for debug, I think I removed it, but it show up